src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,221 @@
+| orig           | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space. |
+| uri or path    | This can be used to point to a specific file.                                                                                 |
+
+### Examples
+
+Preprocessed `bold` NIfTI file in `individual` coordinate space. Please mind
+that in this case `SpatialReference` key is REQUIRED.
+
+```Text
+sub-01/func/sub-01_task-rest_space-individual_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-individual_bold.json
+```
+
+```JSON
+{
+    "SpatialReference": "/sub-01/anat/sub-01_desc-combined_T1w.nii.gz"
+}
+```
+
+Participant cortical thickness GIFTI file in `individual` coordinate space.
+Please mind that in this case `SpatialReference` key is REQUIRED.
+
+```Text
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.shape.gii
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.json
+```
+
+```JSON
+{
+    "SpatialReference": "/sub-01/anat/sub-01_hemi-L_pial.surf.gii"
+}
+```
+
+## Metadata conventions
+
+-   Unless specified otherwise, individual sidecar JSON files and all metadata
+    fields within are optional. However, the appropriate use of these files and
+    pertinent fields is very valuable and thus encouraged. Moreover, for some
+    types of files, there may be one or more required metadata fields, in which
+    case at least one metadata file containing that field must be located
+    somewhere within the file’s hierarchy (per [the Inheritance
+    Principle](../02-common-principles.md#the-inheritance-principle)).
+
+-   When chaining derivative pipelines, any JSON fields that were specified as
+    mandatory in the input files should be propagated forward in the output
+    file’s JSON provided they remain valid. Non-required JSON fields can be
+    propagated, and are highly useful, but it is the pipeline’s responsibility
+    to ensure that the values are still relevant and appropriate to the type of
+    output data.
+
+## File naming conventions
+
+-   Filenames that are permissible for a raw BIDS data type have a privileged
+    status. Any modification of raw files must use a modified filename that does
+    not conflict with the raw filename. Further, any files created as part of a
+    derivative dataset must not match a permissible filename of a valid raw
+    dataset. Stated equivalently, if any filename in a derivative dataset has a
+    name permissible for a raw BIDS data, then that file must be an identical
+    copy of that raw file.
+
+-   Each Derivatives filename MUST be of the form:
+    `<source_keywords>[_keyword-<value>]_<suffix>.<ext>`
+    (where `<value>` could either be an `<index>` or a `<label>` depending on
+    the keyword)
+
+-   When the derivatives chain involves outputs derived from a single raw input,
+    `source_keywords` MUST be the entire source filename, with the ommission of
```

Comment:
`ommission` -> `omission`

Reply:
@Lestropie You can make code suggestions that can be directly applied, e.g.,

```suggestion
    `source_keywords` MUST be the entire source filename, with the omission of
```

See [Adding line comments to a pull request](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) item 6.

Reply:
Cool, had not seen that :+1: